hal/src/stm32f2xx/concurrent_hal.cpp

```diff
@@ -30,6 +30,8 @@
 #include "semphr.h"
 #include <mutex>
 
+// For OpenOCD FreeRTOS support
```

Comment:
extern "C" is only for functions. I doesn't do anything special for variables. A simple extern is sufficient here.